akka-docs-dev/rst/scala/code/docs/stream/StreamTcpDocSpec.scala

```diff
@@ -25,33 +26,41 @@ class StreamTcpDocSpec extends AkkaSpec {
   // silence sysout
   def println(s: String) = ()
 
-  val localhost = new InetSocketAddress("127.0.0.1", 8888)
-
   "simple server connection" ignore {
```

Comment:
[Still disabled](https://github.com/akka/akka/blob/release-2.3-dev/project/AkkaBuild.scala#L82-L84) in release-2.3-dev branch.